src/test/kotlin/com/workos/test/users/UsersTest.kt

```diff
@@ -40,6 +38,46 @@ class UsersTest : TestBase() {
     assertEquals(id, user.id)
   }
 
+  @Test
+  fun authenticateUserWithPasswordReturnsAuthenticationResponse() {
+    val workos = createWorkOSClient()
+
+    val email = "marcelina@foo-corp.com"
+
+    stubResponse(
```

Comment:
Can we add the request body to the stubs as well? That way we are asserting that our payload creation logic is correct as well. Here is an example https://github.com/workos/workos-kotlin/blob/1f28bfd702ce1009fcee414c3864b6bfe36670d4/src/test/kotlin/com/workos/test/auditlogs/AuditLogsTest.kt#L17-L43.

Reply:
done